client/src/components/JurisdictionAdmin/RoundManagement.tsx

```diff
@@ -195,6 +199,18 @@ export const JAFileDownloadButtons = ({
       {auditSettings.auditType === 'BATCH_COMPARISON' ? 'Batch' : 'Ballot'}{' '}
       Retrieval List
     </Button>
+    {auditSettings.auditType === 'BATCH_COMPARISON' &&
+      // TODO: Allow audit admins to download batch tally sheets as well. To do this, update
+      // DownloadBatchTallySheetsButton to make audit admin API calls instead of jurisdiction
+      // manager API calls depending on the user type
+      userType === 'jurisdiction_admin' && (
+        <DownloadBatchTallySheetsButton
+          electionId={electionId}
+          jurisdictionId={jurisdictionId}
+          jurisdictionName={jurisdictionName}
+          roundId={round.id}
+        />
+      )}
```

Comment:
I think audit admins actually can already do this: https://github.com/votingworks/arlo/blob/main/server/api/batches.py#L91-L95. I think I misspoke when giving the Arlo architecture overview to say that the user types had entirely separate APIs - it started that way but as we've given the audit admin more abilities to act like jursidiction admins, that has shifted.

So I think you can just enable it on the frontend for audit admins as well

Reply:
Ooh so I did initially try enabling this for audit admins, but I encountered this permission error

![error-ui](https://user-images.githubusercontent.com/12616928/167771826-0ac0f1ea-0194-4a8d-98d2-85ff935df39b.png)

Taking a closer look, while batch retrieval works fine for both user types, contest retrieval does not. So there's still some API work to be done that feels like it belongs in its own follow-up issue / PR

<img width="400" alt="error" src="https://user-images.githubusercontent.com/12616928/167771833-ab1b61b7-2137-47f5-9b6e-ef931e28359c.png">

Reply:
Fixing that should be as easy as applying this patch:

```diff --git a/server/api/contests.py b/server/api/contests.py
index 35523cf8..bbcbf05d 100644
--- a/server/api/contests.py
+++ b/server/api/contests.py
@@ -223,7 +223,7 @@ def list_contests(election: Election):
 @api.route(
     "/election/<election_id>/jurisdiction/<jurisdiction_id>/contest", methods=["GET"]
 )
-@restrict_access([UserType.JURISDICTION_ADMIN])
+@restrict_access([UserType.AUDIT_ADMIN, UserType.JURISDICTION_ADMIN])
 def list_jurisdictions_contests(
     election: Election, jurisdiction: Jurisdiction,  # pylint: disable=unused-argument
 ):```

Not trying to expand scope here but I do think we generally want AAs to be able to click the same buttons as JAs, and it should be easy to facilitate. If the above doesn't work then I'm ok leaving it for a follow up task.

Reply:
Nice! That did in fact just work! Working on adding an audit admin test now, though, which may take me a sec to figure out